index.html

```diff
@@ -173,30 +173,107 @@ <h2>
     </section>
     <section>
       <h2>
-        <code>pointerlockchange</code> and <code>pointerlockerror</code> Events
+        Pointer Lock and Interfaces
       </h2>
-      <p>
-        Two events are used to communicate pointer lock state change or an
-        error in changing state. They are named <dfn>pointerlockchange</dfn>
-        and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
-        for any reason a <a>pointerlockchange</a> event must be sent.
-      </p>
-      <p>
-        <a>User agents</a> must deliver these events by <a data-lt=
-        "queue a task">queuing a task</a> to <a>fire an event</a> of the
-        appropriate name with its <code>bubbles</code> attribute [[DOM]] set to
-        false to the pointer lock <a>target</a> element's [=Node/node
-        document=].
-      </p>
-      <p class="note">
-        Magnification software increases the size of content on the screen. It
-        uses the mouse to move the magnified point of focus around. When a
-        pointer lock is initiated, the magnification software needs to switch
-        to using the keyboard to move the magnified point of focus around
-        instead. When a pointerlockchange event is fired, web browsers
-        therefore need to make sure the event is communicated to assistive
-        technologies like screen magnifiers.
-      </p>
+      <section>
+        <h3>
+          The <dfn>Pointer Lock State</dfn> Definition
+        </h3>
+          <p><a>Pointer Lock State</a> is the state where a single DOM element, which
+            we will call the <dfn>target</dfn>, receives all mouse events and the cursor
+            is hidden.</p>
+          <p>
+            Once in the <a>Pointer Lock State</a> the <a>user agent</a> must fire all
+            relevant user generated {{MouseEvent}} events (for example:
+            `mousemove`, `mousedown`, `mouseup`, `click`, and `wheel`)
+            [[ui-events]] to the <a>target</a> of pointer lock, and not fire
+            mouse events to other elements. Events that require the concept of
+            a mouse cursor must not be dispatched (for example: `mouseover`,
+            `mouseout`, `drag`, and `drop`).
+          </p>
+          <p>
+            <dl data-dfn-for="PointerLockOptions" data-link-for=
+            "PointerLockOptions">
+              While in the <a>Pointer Lock State</a> if <a>unadjustedMovement</a> is
+              <code>true</code> the event coordinates must not be affected by the
```

Comment:
I think this section shouldn't reference API objects like unadjustedMovement. Instead it should reference concepts like "target" and "unadjusted movement", and then the requestPointerLock() method should translate between API and concepts.

So you could say something like

> If the Pointer Lock State is requested with **unadjusted movement**, then .... Otherwise, the UA must rely...

Reply:
Hmm, I think this comment was overtaken by later parts of the review, which suggest storing a user agent-wide pointer lock options, which does use the `PointerLockOptions` dictionary. In that case I think it's OK to reference API-level concepts, but you should take about the user agent's pointer lock options's unadjustedMovement member being true.

(Note: it cannot be undefined; the Web IDL layer will translate undefined to false.)

Reply:
So I left this mostly the same but I changed the references to unadjusted movement to specifically point out that it is a member of PointerLockOptions.  Some strangeness with the possessive for of the linked word though.